src/utils/sendBackendRequest.js

```diff
@@ -0,0 +1,46 @@
+import { notification } from 'antd'
+
+const PROD_API_URL = process.env.REACT_APP_PROD_API_URL
```

Comment:
Bad naming, imagine if app used 3 or 4 different API-s, which API_URL would be this one of API №1 or №2?
The naming is not semantic, but should be

Reply:
Or change function + file names into more semantic...
Because it's not clear to which backend we are gonna send our request